reporting-engine-worker/src/main/java/org/ihtsdo/termserver/scripting/reports/release/PackageComparisonReport.java

```diff
@@ -61,9 +85,39 @@ public Job getJob() {
 				.withExpectedDuration(40)
 				.build();
 	}
+
+	@Override
+	protected void loadProjectSnapshot(boolean fsnOnly) throws TermServerScriptException, InterruptedException, IOException {
+		//getArchiveManager(true).loadSnapshot(fsnOnly);
+		//Reset the report name to null here as it will have been set by the Snapshot Generator
+		//setReportName(null);
+	}
 	
 	public void runJob() throws TermServerScriptException {
 		//Kick off scripts here
+		boolean isWindows = System.getProperty("os.name").toLowerCase().startsWith("windows");
+
+		ProcessBuilder builder = new ProcessBuilder();
+		if (isWindows) {
+			builder.command("cmd.exe", "/c", "dir");
+		} else {
+			String command = "./" + scriptName + " " + String.join(" ", scriptArguments);
+			builder.command("sh", "-c", command);
+		}
+		builder.directory(new File("reporting-engine-worker/scripts"));//System.getProperty("user.home")));
+
+		try {
+			Process process = builder.start();
+
+			try (BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()))) {
+				reader.lines().forEach(line -> System.out.println(line));
+			}
+
+			int exitVal = process.waitFor();
+			System.out.println("Script execution finished with exit code: " + exitVal);
```

Comment:
The "info" method has issues which I'm going to fix at some point, but it has the advantage that if gives you the standard log4j timestamp, so in the logs as well as having the exit code, we'll have the time that it happened at.

Reply:
I'd like to see you validate the exitCode and fail the report entirely if it's not the expected value (is that 0?)

That helps us detect when something has gone wrong, because the trouble with just outputting the script to STDOUT or GoogleSheet is that we need to scan through it to detect a problem.   It's better to err on the side of blowing up, if the script itself has blown up.    I'm not say blow up unnecessarily, but if the report is non-viable, that would be suffient reason to fail the report entirely.

Reply:
@pgwilliams Old irrelevant code again :-( I use logger to output